core/domain/topic_services_test.py

```diff
@@ -613,11 +613,6 @@ def test_update_topic(self):
             'property_name': topic_domain.TOPIC_PROPERTY_URL_FRAGMENT,
             'old_value': '',
             'new_value': 'url-name'
-        }), topic_domain.TopicChange({
```

Comment:
TOPIC_PROPERTY_THUMBNAIL_FILENAME needs to be removed from this list, because thumbnail_size_in_bytes is calculated based on thumbnail_filename, and if the thumbnail_filename does not exist in the filesystem -- File not found exception is thrown.

Reply:
Fix the filesystem instead -- prepare the test data so that things are valid. 

Add another test to test failure of applying the changelist due to a missing thumbnail image in the file system.

Reply:
Using the [isFile](https://github.com/oppia/oppia/blob/9b0478905fc8037c23536546af7b6819a805d498/core/domain/fs_domain.py#L142) API from fs_domain, we can check if the resource exists in GCS. 
Updated the content accordingly, WDYT ?

Reply:
Updated.